Content.Shared/Delivery/SharedDeliverySystem.cs

```diff
@@ -127,33 +148,38 @@ private void OnGetSpawnerVerbs(Entity<DeliverySpawnerComponent> ent, ref GetVerb
         });
     }
 
-    private bool TryUnlockDelivery(Entity<DeliveryComponent> ent, EntityUid user, bool rewardMoney = true)
+    private bool TryUnlockDelivery(Entity<DeliveryComponent> ent, EntityUid user, bool rewardMoney = true, bool force = false)
     {
         // Check fingerprint access if there is a reader on the mail
-        if (TryComp<FingerprintReaderComponent>(ent, out var reader) && !_fingerprintReader.IsAllowed((ent, reader), user))
+        if (!force && TryComp<FingerprintReaderComponent>(ent, out var reader) && !_fingerprintReader.IsAllowed((ent, reader), user))
             return false;
 
         var deliveryName = _nameModifier.GetBaseName(ent.Owner);
 
-        _audio.PlayPredicted(ent.Comp.UnlockSound, user, user);
+        if (!force)
+            _audio.PlayPredicted(ent.Comp.UnlockSound, user, user);
 
         ent.Comp.IsLocked = false;
         UpdateAntiTamperVisuals(ent, ent.Comp.IsLocked);
 
         DirtyField(ent, ent.Comp, nameof(DeliveryComponent.IsLocked));
 
+        RemComp<SimpleToolUsageComponent>(ent); // we don't want unlocked mail to still be cuttable
```

Comment:
```suggestion
        RemCompDeferred<SimpleToolUsageComponent>(ent); // we don't want unlocked mail to still be cuttable
```
that component is currently raising an event, so we should not remove it until the next tick